### ISSUE TITLE
[Accessibility] Invisible menu items are traversed

### ISSUE THREAD

**Issue by QuncCccccc** [flutter/flutter#185785]
### Steps to reproduce

1. Turn on Talkback, run a11y app
2. Tap the button on right side of the app bar
3. Tap other places outside of the menu panel to dismiss it
4. Swipe to right

### Expected results

The widget lists should be traversed.

### Actual results

The invisible options previously showed up on the menu panel are traversed.

### Code sample

dev/a11y_assessments/lib/main.dart

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

https://github.com/user-attachments/assets/68921c0a-fec0-4c24-88e7-9bc3820c1067

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[Paste your output here]
```

</details>

**Comment by puneetkukreja98** [comment]
@QuncCccccc Thanks for reporting it.
I was able to reproduce it on latest master, using device oneplus nord ce5(CPH2717- Android 16) -  Routing this to team-accessibility for further investigation.
**Screenshot**:

<img width="450" height="700" alt="Image" src="https://github.com/user-attachments/assets/c92a7715-8f82-4403-9cbd-a0ef97eac4a9" />

**Flutter doctor -v**:
```dart
[✓] Flutter (Channel master, 3.44.0-1.0.pre-347, on macOS 26.4 25E246 darwin-arm64, locale en-US) [3.4s]
    • Flutter version 3.44.0-1.0.pre-347 on channel master at /Users/puneetkukreja/flutter-contribution/flutter
    • Upstream repository https://github.com/puneetkukreja98/flutter
    • FLUTTER_GIT_URL = https://github.com/puneetkukreja98/flutter
    • Framework revision 98ab889593 (2 hours ago), 2026-04-29 22:04:34 -0400
    • Engine revision 98ab889593
    • Dart version 3.13.0 (build 3.13.0-79.0.dev)
    • DevTools version 2.57.0
    • Feature flags: enable-web, enable-linux-desktop, enable-macos-desktop, enable-windows-desktop, enable-android, enable-ios, cli-animations, enable-native-assets, enable-swift-package-manager,
      omit-legacy-version-file, enable-lldb-debugging, enable-uiscene-migration, enable-riscv64

[✓] Android toolchain - develop for Android devices (Android SDK version 36.1.0) [2.4s]
    • Android SDK at /Users/puneetkukreja/Library/Android/sdk
    • Emulator version 36.5.10.0 (build_id 15081367) (CL:N/A)
    • Platform android-36.1, build-tools 36.1.0
    • Java binary at: /Library/Java/JavaVirtualMachines/jdk-17.jdk/Contents/Home/bin/java
      This JDK is specified in your Flutter configuration.
      To change the current JDK, run: `flutter config --jdk-dir="path/to/jdk"`.
    • Java version Java(TM) SE Runtime Environment (build 17.0.12+8-LTS-286)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 26.4) [1,456ms]
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 17E192
    • CocoaPods version 1.16.2

[✓] Chrome - develop for the web [6ms]
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Connected device (5 available) [7.9s]
    • CPH2717 (mobile)                    • FQXGT8ZLHE99RKLB          • android-arm64  • Android 16 (API 36)
    • sdk gphone64 arm64 (mobile)         • emulator-5554             • android-arm64  • Android 16 (API 36) (emulator)
    • Puneet’s iPhone (wireless) (mobile) • 00008110-00167CEE0C46401E • ios            • iOS 18.0.1 22A3370
    • macOS (desktop)                     • macos                     • darwin-arm64   • macOS 26.4 25E246 darwin-arm64
    • Chrome (web)                        • chrome                    • web-javascript • Google Chrome 147.0.7727.117

[✓] Network resources [1,157ms]
    • All expected network resources are available.

• No issues found!

```

**Comment by chunhtai** [comment]
duplicate of https://github.com/flutter/flutter/issues/185784
